src/main/java/com/skynest/uitesting/tests/UserRegisterTest.java

```diff
@@ -0,0 +1,21 @@
+package com.skynest.uitesting.tests;
+
+import org.testng.annotations.Test;
+
+public class UserRegisterTest extends BaseTest {
+    @Test
+    public void testUserRegister() {
+        loginPage.clickRegisterLink();
+        sleep(2000);
+        registerPage.enterFirstName(firstName);
+        registerPage.enterLastName(lastName);
+        registerPage.enterEmail(emailAddress);
+        sleep(2000);
+        registerPage.enterPhoneNumber(phoneNumber);
+        registerPage.enterHomeAddress(homeAddress);
+        registerPage.enterPassword(password);
+        registerPage.enterConfirmPassword(confirmPassword);
+        sleep(2000);
+        registerPage.clickRegisterButton();
```

Comment:
also consider combining these anatomical actions (click, enter password etc) into one behaviour based method

something like `fillRegistrationForm("Filip", "1234552")`
or even more fluent

```
fillRegistrationForm()
.withName("Filip")
.withPhone("2134234")
.and()
.submitForm();
```

it would be easier to read and in some cases easier to maintain when changes occur
imagine registration is now splitted into 3 segments in a wizard flow:
-personal details (name + address) -> next
-contacts -> next
-review and submit

you would need to rearrange method calls in each test case defined...
if that logic kept in one place, only some adjustment would be needed there (creating 3 segments for each wizard screen), independently of the data you have entered in the test...